reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Editions.md

```diff
@@ -240,7 +240,10 @@ to catch possible behavioral differences between editions. For this you must sti
 
 [Pester]: https://github.com/pester/Pester/wiki/Pester
 [PSScriptAnalyzer]: https://github.com/PowerShell/PSScriptAnalyzer
-[`PSUseCompatibleCommands`]: https://github.com/PowerShell/PSScriptAnalyzer/blob/master/RuleDocumentation/UseCompatibleCommands.md
-[`PSUseCompatibleTypes`]: https://github.com/PowerShell/PSScriptAnalyzer/blob/master/RuleDocumentation/UseCompatibleTypes.md
+[`PSUseCompatibleCommands`]:
+https://github.com/PowerShell/PSScriptAnalyzer/blob/master/RuleDocumentation/UseCompatibleCommands.md
```

Comment:
These are definitions for a reference links. The label and the url need to be on the same line.